docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+app:
+  build: .
+  volumes:
+    - ./tock:/tock
+  links:
+    - db
+  working_dir: /tock
+  entrypoint: python /tock/docker_entrypoint.py
+  environment:
+    - DATABASE_URL=postgres://tock_user@db/tock
+  command: python manage.py runserver 0.0.0.0:8000
+db:
+  image: postgres:9.4
+  environment:
+    - POSTGRES_DB=tock
+    - POSTGRES_USER=tock_user
+web:
+  image: nginx
```

Comment:
Um, I'm not actually sure how #333 even worked without adding this nginx proxy.  The tests run fine without it, but visiting the server without the nginx proxy just results in a `403 Forbidden` for me. It seems the dev nginx configuration adds the appropriate headers to make the app think it's always authenticated.